api/lib/infrastructure/utils/string-utils.js

```diff
@@ -42,11 +42,37 @@ function normalize(str) {
   return [...strUpper].filter((char) => Boolean(char.match(/[0-9A-Z]/))).join('');
 }
 
+function capitalize(str) {
+
+  const charArray = str.toLowerCase().split('');
+
+  charArray.forEach((char, index) => {
+    if (index === 0) {
+      return charArray[index] = charArray[index].toUpperCase();
+    }
+
+    if (_isCharADashOrSpace(char)) {
+      return charArray[index + 1] = charArray[index + 1].toUpperCase();
+    }
+
+    return charArray;
+  });
+
+  return charArray.join('');
+}
+
 function _removeNonCanonicalChars(str) {
   return str.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
 }
 
+function _isCharADashOrSpace(char) {
+  const separators = ['-', ' '];
+
+  return new RegExp(char).test(separators);
```

Comment:
Si `char` est `.` par exemple, cette fonction va renvoyer `true` il me semble.
```suggestion
  return /[- ]/.test(char);
```
ou sinon
```suggestion
  return separators.indexOf(char) >= 0;
```